example/src/App.tsx

```diff
@@ -53,6 +53,20 @@ function App() {
                         >
                             example_protected.pdf (Password: 123456)
                         </button>
+
+                        <input
+                            className="button"
```

Comment:
shouldnt that be "input" ?

Reply:
I don't have separate styles for input. It should look like the other buttons.

Reply:
ok, lgtm

Reply:
Thank you!